libcst/metadata/scope_provider.py

```diff
@@ -41,11 +56,16 @@ def record_access(self, access: Access) -> None:
 
     @property
     def accesses(self) -> Tuple[Access, ...]:
+        """Return all accesses of the assignment."""
         # we don't want to publicly expose the mutable version of this
         return tuple(self.__accesses)
 
 
 class Assignment(BaseAssignment):
+    """An assignment records the name, CSTNode and its accesses."""
+
+    #: The node of assignment, it could be a :class:`~libcst.Import`, :class:`ImportFrom`,
```

Comment:
Maybe we should link to Python language specification for variable assignment? This is not immediately clear as to why an Assignment handles Import/ImportFrom, FunctionDef and ClassDef.

Reply:
The Assignment naming here is different than the regular Assignment in Python. Maybe we should have a different naming to make it more clear. E.g. "Definition"

Reply:
After discussed with @bgw, Assignment is a better naming here than other candidates. In Python, the concept of declaration and assignment are mixed together. A name could be assigned multiple times and Declaration usually mean the only one declaration. Definition can refer to FunctionDef or ClassDef but it's not directly linked to assignments.
So we decided to keep the name for now.